oss/interval_tree/LICENSE

```diff
@@ -0,0 +1,19 @@
+Copyright (c) 2011 Erik Garrison
+

```

Comment:
license is discovered by component governance; no need to include.

please put a snipept inside the NOTICE.MD file in the root of this repo! follow the existing format :smile: